src/main/webapp/app/entities/assessment-note.model.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseEntity } from 'app/shared/model/base-entity';
+import { User } from 'app/core/user/user.model';
+
+export class AssessmentNote implements BaseEntity {
+    public id?: number;
```

Comment:
This is already present in `BaseEntity`

Reply:
Are you referring to the ID? Because I still have to include it even though it's part of `BaseEntity`.

Reply:
The `id` is also present for other model classes. The compiler does not accept it if the interface is implemented without having this attribute.

Reply:
Yes, I was talking about the id. Does it not work just via `implements`?

Reply:
No, there probably is some difference how interface attributes are handled compared to abstract base classes.